packages/@expo/log-box/src/logbox-rn-polyfill.tsx

```diff
@@ -0,0 +1,200 @@
+import { isRunningInExpoGo } from 'expo';
+import React, { useMemo } from 'react';
+import { View, DevSettings, Platform, Clipboard, type Modal as ModalInterface } from 'react-native';
+// @ts-ignore
+import * as LogBoxData from 'react-native/Libraries/LogBox/Data/LogBoxData';
+// @ts-ignore
+import RCTModalHostView from 'react-native/Libraries/Modal/RCTModalHostViewNativeComponent';
+
+import { LogBoxLog } from './Data/LogBoxLog';
+import LogBoxPolyfillDOM from './logbox-dom-polyfill';
+import { getBaseUrl } from './utils/devServerEndpoints';
+
+const Modal = RCTModalHostView as typeof ModalInterface;
+
+const Colors = {
+  background: '#111113',
+};
+
+function LogBoxRNPolyfill(props: {
+  onDismiss: (index: number) => void;
+  onMinimize: () => void;
+  onChangeSelectedIndex: (index: number) => void;
+  logs: any[];
+  selectedIndex: number;
+}) {
+  const logs = React.useMemo(() => {
+    return props.logs.map((log) => {
+      return {
+        symbolicated: log.symbolicated,
+        symbolicatedComponentStack: log.symbolicatedComponentStack,
+        componentCodeFrame: log.componentCodeFrame,
+        level: log.level,
+        type: log.type,
+        message: log.message,
+        stack: log.stack,
+        category: log.category,
+        componentStack: log.componentStack,
+        componentStackType: log.componentStackType,
+        codeFrame: log.codeFrame,
+        isComponentError: log.isComponentError,
+        extraData: log.extraData,
+        count: log.count,
+      };
+    });
+  }, [props.logs]);
+
+  const [open, setOpen] = React.useState(true);
+  const bundledLogBoxUrl = getBundledLogBoxURL();
+
+  const closeModal = (cb: () => void) => {
+    setOpen(false);
+    setTimeout(cb, Platform.select({ ios: 500, default: 0 }));
+  };
+
+  const onMinimize = () => closeModal(props.onMinimize);
+  const onDismiss = props.onDismiss;
+
+  const LogBoxWrapper = useMemo(
+    () =>
+      Platform.OS === 'ios'
+        ? ({ children }: { children: React.ReactNode }) => {
+            return (
+              <Modal
+                animationType="slide"
+                presentationStyle="pageSheet"
+                visible={open}
+                onRequestClose={onMinimize}>
+                {children}
+              </Modal>
+            );
+          }
+        : ({ children }: { children: React.ReactNode }) => <>{children}</>,
+    []
+  );
+
+  return (
+    <LogBoxWrapper>
+      <View
+        style={{
+          backgroundColor: Platform.select({ default: undefined, ios: Colors.background }),
+          pointerEvents: 'box-none',
+          top: 0,
+          flex: 1,
+        }}
+        collapsable={false}>
+        <LogBoxPolyfillDOM
+          platform={process.env.EXPO_OS}
+          devServerUrl={getBaseUrl()}
+          dom={{
+            useExpoDOMWebView: true,
```

Comment:
Is this better than standard webview?

Reply:
I opted for the expo dom webview since it's smaller, and this will be pulled into every expo app.